src/aluno/PerfilPage.tsx

```
@@ -135,12 +135,12 @@ export function PerfilPage() {
     }
 
     function exportEquipeCsv(projetos: Projeto[]) {
-        const rows: string[] = ['Projeto,Título,Nome,Email']
+        const rows: string[] = ['Título,Nome,Email']
 
         projetos.forEach((projeto) => {
             const equipe = projeto.equipeExecutora || []
             equipe.forEach((membro) => {
-                const linha = `"${projeto.id}","${projeto.titulo}","${membro.nome}","${membro.email}"`
+                const linha = `""${projeto.titulo}","${membro.nome}","${membro.email}"`
```

ESLint

[error] Replace `················` with `········`

(prettier/prettier)